packages/cli/src/commands/init.ts

```diff
@@ -726,3 +740,16 @@ async function setupEnvironmentVariable(
     logger.success(`✅ Added ${variableName}=${renderer(value)} to ${fileName}`);
   }
 }
+
+async function detectNextJsVersion(projectPath: string): Promise<number> {
+  const packageFile = pathModule.join(projectPath, "package.json");
+  const packageData = await fs.readFile(packageFile, "utf8");
+  const packageJson = JSON.parse(packageData);
```

Comment:
You should use the `readPackageJson()` function. It returns a typed `PackageJson` object, or undefined if the files doesn't exist.

Reply:
I'll do this instead